src/app/grid/grid-paste/grid-paste-sample.component.html

```diff
@@ -1,8 +1,8 @@
 <div class="grid__wrapper">
     <div class="grid__menu">
-        <button igxButton="raised" (click)="toggleDropDown($event)">Paste Options</button>
-        <igx-drop-down (onSelection)="selectionChange($event)">
-            <igx-drop-down-item *ngFor="let item of comboData">
+        <button igxButton="raised" [igxToggleAction]="dropdown" [igxDropDownItemNavigation]="dropdown">Paste Options</button>
+        <igx-drop-down #dropdown (onClosing)="changePasteMode()">
```

Comment:
fairly certain there's selectionChanged event you can use and skip the viewchild :)